http/client/src/main/java/org/eclipse/rdf4j/http/client/SharedHttpClientSessionManager.java

```diff
@@ -56,7 +56,8 @@
 
 	public SharedHttpClientSessionManager() {
 		final ThreadFactory backingThreadFactory = Executors.defaultThreadFactory();
-		this.executor = Executors.newScheduledThreadPool(0, new ThreadFactory() {
+		int cores = (System.getProperty("org.rdf4j.client.executors.jdkbug") != null) ? 1 : 0;
```

Comment:
I’m comfortable with this kind of change if you add a nice big juicy paragraph-long comment explaining what the value should be and which bug is causing the issue with a link to the jdk bug. 

Also. Can you pull “cores” out as a static final field? We don’t need to support this dynamically changing.

Reply:
Ok, good point, I've moved core to static final and added a few lines with the link to the openjdk bug report.